make/common/modules/LauncherCommon.gmk

```diff
@@ -70,19 +66,25 @@ define SetupBuildLauncherBody
     $1_OPTIMIZATION := LOW
   endif
 
-  ifeq ($$($1_MAIN_MODULE), )
```

Comment:
Why was this check removed? Is MAIN\_MODULE no longer needed?

Reply:
It is never overridden. I don't know what would make us override it either, seems just weird.